### ISSUE TITLE
Implement descriptions for all Ant tasks

### ISSUE THREAD

**Issue by lewismc** [joshua-decoder/joshua#188]
This issue should produce the following when one invokes *ant -projecthelp*
```
lmcgibbn@LMC-032857 /usr/local/joshua(master) $ ant -projecthelp
Buildfile: /usr/local/joshua/build.xml

Main targets:

 all                 --> Default target Java build tasks
 check-joshua-home   --> Check for environment variables
 clean               --> Clean (remove) all build directories
 clean-java          --> Delete just the java files
 clean-thrax         --> Delete just the thrax files
 compile-tests       --> Compile the unit test code
 devel-clean         --> Delete *all* generated files, including files and directories not tracked by git
 distclean           --> EXPERIMENTAL: Delete *all* generated files
 documentation       --> Compile the Javadocs and Markdown documentation into web pages
 download-hadoop     --> Download the Hadoop software
 download-thrax      --> Download the Thrax software
 git-version         --> Store latest git commit ID
 giza                --> 'Make' the giza software in scripts/training/giza-pp/
 init                --> Check for environment variables, retrieve dependencies with ivy, download Thrax and Hadoop is required
 init-ivy            --> Try to load and initialize Ivy here from Ivy Home
 jar                 --> Create a JAR file of compiles classes
 java                --> Compile the Java code
 javadoc             --> Compile the Javadocs into web pages
 kenlm               --> 'Make' the kenlm sofrtware in ${src}/joshua/decoder/ff/lm/kenlm/
 parallelize         --> 'Make' the parallelize software in scripts/training/parallelize/
 release             --> Create a versioned release
 resolve             --> retrieve dependencies with ivy
 set-java-home       --> Prompt user to set JAVA_HOME environment variable
 set-joshua-version  --> Prompt user to set Joshua version environment variable
 source-jar          --> Create a JAR file of the source code
 test                --> Run tests
 testng              --> Run the unit tests
 thrax               --> Build Thrax
 tree_visualizer     --> Runs the Tree Visualizer software in examples/tree_visualizer/tree_visualizer.jar
Default target: all
```

**Comment by mjpost** [comment]
Closed with commit 0a2a17b288734a2e32c3e7d4b5bd26b23d23c4d9.

**Comment by lewismc** [comment]
Thanks Matt

On Friday, March 13, 2015, Matt Post <notifications@github.com> wrote:

> Closed #188 <https://github.com/joshua-decoder/joshua/issues/188>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joshua-decoder/joshua/issues/188#event-251889961>.
>


-- 
*Lewis*
